doc/installing/development-win.md

```diff
@@ -0,0 +1,106 @@
+# Installing the OpenQuake Engine for development using Python 3.6 on Windows (provisional)
+
+## Prerequisites
+
+Knowledge of [Python](https://www.python.org/) (and its virtual environments), [git](https://git-scm.com/) and [software development](https://xkcd.com/844/) are required.
```

Comment:
I would add a link to https://docs.python.org/3.6/tutorial/venv.html

Reply:
Done and also backported to `development.md`